code/modules/projectiles/guns/projectile/pistol.dm

```diff
@@ -180,13 +183,17 @@
 
 	var/global/list/ammo_types = list(
 		/obj/item/ammo_casing/a357              = ".357",
+		/obj/item/ammo_casing/c9mmf             = "9mm",
+		/obj/item/ammo_casing/c45f              = ".45",
 		/obj/item/ammo_casing/a12mm             = "12mm",
 		/obj/item/ammo_casing/shotgun           = "12 gauge",
 		/obj/item/ammo_casing/shotgun           = "12 gauge",
 		/obj/item/ammo_casing/shotgun/pellet    = "12 gauge",
 		/obj/item/ammo_casing/shotgun/pellet    = "12 gauge",
 		/obj/item/ammo_casing/shotgun/pellet    = "12 gauge",
 		/obj/item/ammo_casing/shotgun/beanbag   = "12 gauge",
+		/obj/item/ammo_casing/shotgun/stunshell = "12 gauge",
+		/obj/item/ammo_casing/shotgun/flash     = "12 gauge",

```

Comment:
You are adding the calibers that were removed on purpose from the heister's zipguns back again.